README.md

```diff
@@ -1 +1,44 @@
 # sonarqube-demo
+
+ Download sonarqube zip file and extract it.
+ Open directory -> bin -> linux -> open terminal.
+ Run the command: ./sonar.sh start
+ 
+# checking sonarqube in local
```

Comment:
Let's not use steps as h1 headers. Also the documentation should always say "Do ..." "Check ..." instead of "Doing.." , "Checking.."  because we are providing the consumer of this codebase with a guide

Syntax -> https://guides.github.com/features/mastering-markdown/
Example documentation -> https://github.com/caarmen/SunriseSunset